tests/integration/aliases/rename-component/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'application'
+}
+
+group 'com.pulumi.example'
+version '0.0.1-SNAPSHOT'
+
+var gprUser = project.findProperty("gpr.user")  ?: System.getenv("GPR_USER")
+var gprToken = project.findProperty("gpr.token") ?: System.getenv("GPR_TOKEN")
+
+repositories {
+    mavenLocal()
+    maven { // The google mirror is less flaky than mavenCentral()
+        url("https://maven-central.storage-download.googleapis.com/maven2/")
+    }
+    mavenCentral()
+
+    if (gprUser) {
+        repositories {
+            maven {
+                url = uri("https://maven.pkg.github.com/pulumi/pulumi-java")
+                credentials {
+                    username = gprUser
+                    password = gprToken
+                }
+            }
+        }
+    }
+}
+
+var pulumiJavaSdkVersion = System.getenv("PULUMI_JAVA_SDK_VERSION") ?: "0.5.2"
```

Comment:
this should probably be:
`var pulumiJavaSdkVersion = System.getenv("PULUMI_JAVA_SDK_VERSION") ?: "0.0.1"`

we currently have a hack that uses `0.0.1` as a special version used in dev

Reply:
You might have been looking at the old files. This was reverted back to `0.01`.